.jenkins/pipelines/Azure/Nightly/Packaging.Jenkinsfile

```diff
@@ -27,7 +27,7 @@ def LinuxPackaging(String version, String build_type, String lvi_mitigation = 'N
                            cpack
                            ctest --output-on-failure --timeout ${CTEST_TIMEOUT_SECONDS}
                            """
-                oe.Run("clang-7", task)
```

Comment:
Not sure how easy it would be to parametrise this all the way through (or if it even makes sense, would two versions of clang ever be used at the same time), but it could help keep things in sync.

Reply:
This is a good point, the changes if there were to be one would need to be here though: https://github.com/openenclave/openenclave-ci/pull/22/files.

To the best of my knowledge, we are planning to deprecate clang-7 and then moving to 10 in the new year. How about I parameterize and leave the default to clang-8?

Reply:
That sounds good to me, I can't think of any reason to use more than one variant of clang at once, and this will make future updates easier I would think.